app/src/views/CountryNsOverviewContextAndStructure/NationalSocietyLocalUnits/LocalUnitViewModal/styles.module.css

```diff
@@ -0,0 +1,3 @@
+.local-unit-view-children-container {
+    max-height: 36rem;
```

Comment:
Are we trying to set `max-height`? Lets use `min` function to ensure that it does not go beyond `100vh` for smaller screens